Add cron character explanation and give examples of different schedules (#17)
fermyon/spin-trigger-cron: README.md

## Trigger Configuration

The trigger type is `cron`, and there are no application-level configuration options. 

The following options are available to set in the [[trigger.cron]] section:

| Name                  | Type             | Required? | Description |
|-----------------------|------------------|-----------|-------------|
| `component`           | string or table  | required  | The component to run on the schedule given in `cron_expression`. (This is the standard Spin trigger component field.) |
| `cron_expression`     | string           | required  | The `cron` expression describes the schedule for executing the component. |

### Cron Expression Fields

The `cron_expression` format for scheduling is as follows:

```text

> sec   min   hour   day of month   month   day of week   year

Reviewer: This isn't actually part of the expression is it?

Author: Ah, yes. I have updated so that we refer to the `sec`, `min` etc. as fields. I also use a bit of ascii-art inspired text to show each of the fields.

*     *     *      *              *       *             *
```

### Cron Expression Characters

> The `*` indicates that every value applies; i.e., if `sec` is set to `*`, then every second will trigger execution.

Reviewer: ```suggestion
The `*` indicates that every value applies. For example. if `sec` is set to `*`, then every second will trigger execution.
```

(this is the rubric in the other bullet points and it reads better to me)

The `/` indicates increments. For example, if `sec` is set to `0/15`, then starting at `0`, the trigger will be executed every 15 seconds.

> The `,` separates values. For example, if `sec` is set to `2,8`, then the trigger will execute only on the 2nd and 8th seconds of every minute.

Reviewer: Maybe:

```suggestion
The `,` indicates a list of values. For example, if `sec` is set to `2,8`, then the trigger will execute only on the 2nd and 8th seconds of every minute.
```

> The `-` indicates range, i.e., if the `sec` is set to `5-10`, then the trigger will execute only on the 5th, 6th, 7th, 8th, 9th, and 10th seconds of each minute.

Reviewer: ```suggestion
The `-` indicates range. For example, if the `sec` is set to `5-10`, then the trigger will execute only on the 5th, 6th, 7th, 8th, 9th, and 10th seconds of each minute.
```

> The `0` indicates no execution. If the `sec` is set to `0`, then the trigger can only execute on higher field values such as `min`, `hour`, etc. The lowest second increment is 60 (one minute)

Author: ```suggestion
The `0` indicates no execution. If the `sec` is set to `0`, then the trigger can only execute on higher field values such as `min`, `hour`, etc. The lowest second increment is 60 (one minute).
```
